standard.tex

```diff
@@ -66,15 +69,41 @@ \subsection{General Wiring \& Assembly}
 \item Keep good isolation. That means standoffs and that means decent creepage distances.
 \item When using shielded cable, ensure the shield is properly terminated and grounded. Shields should only be grounded at one end of the cable.
 \end{itemize}
+%end of wiring and assembly subsection -- }}}
 
+%start of software subsection -- {{{
 \subsection{Software}
 \begin{itemize}
 \item If your system has any software anywhere, that software should be unit tested.
 \item If your system requires a computer program to be used, that program counts as mission critical. It should run on at least two machines and at least two people should be trained to use it.
 \item All mission critical software should be expressible (and accompanied by) less than 100 lines of pseudocode.
 \item All software should use a VCS (probably git), and all final versions of your software should go with your development logs (see Documentation Standards) on the drive.
 \end{itemize}
+%end of software subsection -- }}}
 
+%start of pcb subsection -- {{{
+\subsection{PCB design}
+\begin{itemize}
+\item Watch this EEVblog video: \url{https://www.youtube.com/watch?v=JrH_itjMDjo}. It's like an hour and a half long, but well worth watching.
+\item Given the choice, please draft your schematics and board layouts in KiCad. It's free (GPLv3), used by the rest of the team, and runs on all platforms. \url{http://kicad-pcb.org/}
+\item if your PCBs are meant to be screwed onto something, put those mounting holes on your layout, and if at all possible, use M3 screws (for standardization purposes, if you have a space restriction, smaller screws are fine).
```

Comment:
"If your PCBs are meant to be screwed onto something, put mounting holes on your layout, and if at all possible, use M3 screws (for standardization purposes. If you have a space restriction, smaller screws are fine)."

Reply:
changed to your version